compute/cloud-client/src/main/java/compute/ipaddress/AssignStaticExternalNewVmAddress.java

```diff
@@ -0,0 +1,150 @@
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.NetworkInterface;
+import java.io.IOException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class AssignStaticExternalNewVmAddress {
+
+  public static void main(String[] args)
+          throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Project ID or project number of the Cloud project you want to use.
+    String projectId = "your-project-id";
+    // Instance ID of the Cloud project you want to use.
+    String instanceId = "your-instance-id";
+    // name of the zone to create the instance in. For example: "us-west3-b"
+    String zone = "your-zone-id";
+    // machine type of the VM being created. This value uses the
+    // following format: "zones/{zone}/machineTypes/{type_name}".
+    // For example: "zones/europe-west3-c/machineTypes/f1-micro"
+    String machineType = String.format("zones/%s/machineTypes/{your-machineType-id}", zone);
+    // boolean flag indicating if the instance should have an external IPv4 address assigned.
+    boolean externalAccess = true;
+    // external IPv4 address to be assigned to this instance. If you specify
+    // an external IP address, it must live in the same region as the zone of the instance.
+    // This setting requires `external_access` to be set to True to work.
+    String externalIpv4 = "your-externalIpv4-id";
+
+    assignStaticExternalNewVmAddress(projectId, instanceId, zone,
+            externalAccess, machineType, externalIpv4);
+  }
+
+  // Create a new VM instance with assigned static external IP address.
+  public static Instance assignStaticExternalNewVmAddress(String projectId, String instanceName,
+                                                          String zone, boolean externalAccess,
+                                                          String machineType, String externalIpv4)
+          throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    String sourceImage;
+    try (ImagesClient imagesClient = ImagesClient.create()) {
+      sourceImage = imagesClient.getFromFamily("debian-cloud", "debian-10").getSelfLink();
+    }
+    AttachedDisk attachedDisk = buildAttachedDisk(sourceImage, zone);
+    return createInstance(projectId, instanceName, zone,
+            attachedDisk, machineType, externalAccess, externalIpv4);
+  }
+
+  private static AttachedDisk buildAttachedDisk(String sourceImage, String zone) {
+    AttachedDiskInitializeParams initializeParams = AttachedDiskInitializeParams.newBuilder()
+            .setSourceImage(sourceImage)
+            .setDiskSizeGb(10)
+            .setDiskType(String.format("zones/%s/diskTypes/pd-standard", zone))
+            .build();
+    return AttachedDisk.newBuilder()
```

Comment:
```suggestion
    
    return AttachedDisk.newBuilder()
```

Reply:
Applies to multiple places. Add a new line wherever appropriate to improve readability.